isis/src/base/objs/Cube/Cube.cpp

```diff
@@ -98,6 +98,10 @@ namespace Isis {
    */
   void Cube::fromIsd(const FileName &fileName, Pvl &label, nlohmann::json &isd, QString access) {
     fromLabel(fileName, label, access);
+
+    if (isd.contains("line_scan_rate")) {
+      attachLineScanTableFromIsd(isd);
+    }
```

Comment:
I am a little worried that this attaches the ```line_scan_rate``` from any ISD. All of our line scanner ISDs have the attribute

Reply:
True. Updated check.